lib/patch-series.ts

```diff
@@ -178,42 +180,96 @@ export class PatchSeries {
             basedOn,
             cc,
             coverLetter,
-        } = PatchSeries.parsePullRequestDescription(pullRequestDescription);
+        } = await PatchSeries.parsePullRequest(workDir,
+                                               pullRequestTitle,
+                                               pullRequestBody,
+                                               senderName, senderEmail);
 
         if (basedOn && !revParse(basedOn, workDir)) {
             throw new Error(`Cannot find base branch ${basedOn}`);
         }
 
         const publishToRemote = undefined;
 
-        const project = await ProjectOptions.get(workDir, headCommit, cc || [],
+        const project = await ProjectOptions.get(workDir, headCommit, cc,
                                                  basedOn, publishToRemote,
                                                  baseCommit);
 
-        const wrapCoverLetterAtColumn = 76;
         return new PatchSeries(notes, options, project, metadata,
                                rangeDiff,
-                               md2text(coverLetter, wrapCoverLetterAtColumn),
+                               coverLetter,
                                senderName);
     }
 
-    protected static parsePullRequestDescription(description: string): {
+    protected static async parsePullRequest(workDir: string,
+                                            prTitle: string,
+                                            prBody: string,
+                                            senderName?: string,
+                                            senderEmail?: string):
+    Promise <{
+        coverLetter: string,
+        basedOn?: string,
+        cc: string[],
+    }> {
+        // Replace \r\n with \n to simplify remaining parsing.
+        // Note that md2text() in the end will do the replacement anyway.
```

Comment:
A very good point.  Is the `\r\n` to `\n` needed when we are now more aware of the issue and can code for it.

Reply:
Yes, otherwise we'll have two bad choices:
1. Write extra code to preserve `\r\n` - see previous discussion: https://github.com/gitgitgadget/gitgitgadget/pull/163#discussion_r355656802
2. Ignore that and mix `\r\n` with `\n` in `parsePullRequestBody()` - that's dirty on its own, but also relies on hidden knowledge that _current_ outside caller drops `\r\n` later anyway, which is even more dirty.

So replacing with `\n` is best.